precinct_splits.xsd

```diff
@@ -0,0 +1,64 @@
+  <xs:element name="vip_object">
+    <xs:complexType>
+      <xs:choice maxOccurs="unbounded">
+      <xs:element name="precinct">
+          <xs:complexType>
+            <xs:sequence>
+              <xs:element minOccurs="1" maxOccurs="1" name="name" type="xs:string" />
+              <xs:element minOccurs="0" maxOccurs="1" name="number" type="xs:string" />
+              <xs:element minOccurs="0" maxOccurs="1" name="precinct_split_name" type="xs:string" />
+              <xs:element minOccurs="1" maxOccurs="1" name="locality_id" type="xs:string" />
+              <xs:element minOccurs="0" maxOccurs="unbounded" name="electoral_district_id" type="xs:string" />
+              <xs:element minOccurs="0" maxOccurs="1" name="ward" type="xs:string" />
+              <xs:element minOccurs="0" maxOccurs="1" name="mail_only" type="yesNoEnum" />
+              <xs:element minOccurs="0" maxOccurs="unbounded" name="polling_location_id" type="xs:string" />
+              <xs:element minOccurs="0" name="ballot_style_image_url" type="xs:string" />
+              <xs:element minOccurs="0" maxOccurs="unbounded" name="external_identifier_id" type="xs:string" />
+            </xs:sequence>
+            <xs:attribute name="id" type="xs:string" use="required" />
+          </xs:complexType>
+        </xs:element>
+        <xs:element name="street_segment">
+          <xs:complexType>
+            <xs:sequence>
+              <xs:element name="start_house_number" type="xs:integer" />
+              <xs:element name="end_house_number" type="xs:integer" />
+              <xs:element name="odd_even_both" type="oebEnum" />
+              <xs:element name="non_house_address" type="detailAddressType" />
+              <xs:element name="unit_number" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
+              <xs:element name="precinct_id" type="xs:string" />
+            </xs:sequence>
+            <xs:attribute name="id" type="xs:string" use="required" />
+          </xs:complexType>
+        </xs:element>
+        <xs:element name="external_identifier">
+            <xs:complexType>
+              <xs:all>
+                <xs:element type="identifierType" name="type"/>
+                <xs:element type="xs:string" minOccurs="0" name="othertype"/>
+                <xs:element type="xs:string" minOccurs="0" name="value"/>
+              </xs:all>
+              <xs:attribute name="id" type="xs:string" use="required" />
+            </xs:complexType>
+        </xs:element>
+      </xs:choice>
+    </xs:complexType>
+  </xs:element>
+  <xs:simpleType name="identifierType">
+  <xs:restriction base="xs:string">
+    <xs:enumeration value="ocd-id"/>
+    <xs:enumeration value="FIPS"/>
+    <xs:enumeration value="national"/>
+    <xs:enumeration value="state"/>
+    <xs:enumeration value="local"/>
+    <xs:enumeration value="OTHER"/>
```

Comment:
@pstenbjorn I know it's on me to create the coding style guide (working on it), but, following some of the 1622 conventions, which will start as the basis for the guide, the enumerations should be lowercase.